core/src/main/scala/org/dbpedia/extraction/mappings/CommonsResourceExtractor.scala

```diff
@@ -0,0 +1,49 @@
+package org.dbpedia.extraction.mappings
+
+import scala.language.reflectiveCalls
+import org.dbpedia.extraction.destinations.{DBpediaDatasets, Quad}
+import org.dbpedia.extraction.ontology.Ontology
+import org.dbpedia.extraction.util.Language
+import org.dbpedia.extraction.wikiparser.{PageNode, TextNode}
+
+import scala.collection.mutable.ArrayBuffer
+
+/**
+ * Links non-commons DBpedia resources to their DBpedia Commons counterpart using owl:sameAs.
+ * This requires the the Wikipedia page to contain a {{Commons}} template.
```

Comment:
can you also extend this a bit and show an example input template with values and the generated triples?